ovos_number_parser/numbers_mwl.py

```
@@ -1,291 +1,237 @@
-}
-
-
-def get_number_map(scale: Scale = Scale.LONG):
-    return {
-        **_NUMBERS_BASE,
-        **{s_name: val for val, s_name, _ in _SCALES[scale]},
-        **{p_name: val for val, _, p_name in _SCALES[scale]}
+from ovos_number_parser.util import Scale, GrammaticalGender, DigitPronunciation, word_tokenize
+
+
+from ovos_number_parser.util import (Scale, GrammaticalGender, DigitPronunciation,
```

Ruff (0.14.7)

Redefinition of unused `Scale` from line 3

Remove definition: `Scale`

(F811)